### DISCUSSION TITLE
Version change to prefixed with "v"

### DISCUSSION THREAD

**Discussion by jokay** [renovatebot/renovate#10549] [Request Help] [answered]
This happened using renovate version **25.49.3**.

The proposed update switched from a version without prefixed "v" to a version with prefixed "v".

![traefik-version](https://user-images.githubusercontent.com/18613935/123056319-14937600-d407-11eb-80ed-9b4fddcd1134.png)

We want to keep the version without the prefixed "v", e.g. [this](https://hub.docker.com/layers/traefik/library/traefik/2.4.9/images/sha256-bfba2ddb60cea5ebe8bea579a4a18be0bf9cac323783216f83ca268ce0004252?context=explore) one.

**Comment by viceice** [comment]
You can workaround it with a package rule and allowedVersions

```json
{
  "packageRules": [
    {
      "matchPackageNames": ["docker.io/traefik"],
      "allowedVersions": "/^[0-9]+\\.[0-9]+\\.[0-9]$/"
    }
  ]
}
```

https://docs.renovatebot.com/configuration-options/#allowedversions

Bug already reported as #4339

**Reply by jokay** [reply]
Sure this is possible, but why it's necessary as it worked before?

**Reply by viceice** [reply]
Nope, it never worked. It simply depends on the docker tag order.

**Comment by jokay** [comment, marked as answer]
[This](https://github.com/renovatebot/renovate/discussions/10549#discussioncomment-909140) is the answer, it depends on the docker tag order.
